nixos/modules/programs/shadow.nix

```
@@ -5,6 +5,18 @@ let
 in
 {
   options = with lib.types; {
```

```suggestion
  options = {
```
While we are at it, we should drop this and just write lib.types.